File: 05-multiplication-table/mult_table.lua
-- Lua program to generate multiplication tables
-- Created by M.V.Harish Kumar on 18-06-2023

function GenerateTables(number, times)
    print(number .. " Tables\n")
    for t = 1, times do
        print(string.format("%d × %d = %d", number, t, number*t))
    end
end

io.write("Enter the no. to generate tables: ")
local n = tonumber(io.read("*n"))
io.write("Enter the no. till which to generate: ")
local t = tonumber(io.read("*n"))
GenerateTables(n, t)

File: 05-multiplication-table/MultTable.cs
// C# program to generate multiplication tables
// Created by M.V.Harish Kumar on 06-01-2024
using System;

namespace MultTable
{
    class MultTable
    {
        static void GenTable(int n, int t)
        {
            Console.WriteLine($"{n} Tables\n");
            for (int i = 1; i <= t; i++)
            {
                Console.WriteLine($"{n} × {i} = {n*i}");
            }
        }

        static void Main(string[] args)
        {
            Console.Write("Enter the no. to generate tables: ");
            int num = Convert.ToInt32(Console.ReadLine());
            Console.Write("Enter the no. till which to generate: ");
            int times = Convert.ToInt32(Console.ReadLine());
            GenTable(num, times);
        }
    }
}